src/translations/be-BY.json

```diff
@@ -177,6 +177,7 @@
   "cboard.components.CommunicatorDialog.boardInfoAuthor": "Савет Аўтар",
   "cboard.components.CommunicatorDialog.boardInfoTiles": "колькасць плітак",
   "cboard.components.CommunicatorDialog.boardInfoId": "Савет ID",
+  "cboard.components.CommunicatorDialog.boardInfoUrl": "Board URL",
```

Comment:
check translations, did you perform the pre-translation before building?

Reply:
i forgot it